rero_mef/utils.py

```
@@ -186,16 +186,20 @@ def _next_response(self):
         if access_token:
             params["accessToken"] = access_token
 
-        count = 0
-        while count < 5:
+        last_error = None
+        # Retry up to 5 times if harvesting fails (network issues, server errors)
+        for count in range(5):
             try:
                 self.oai_response = self.sickle.harvest(**params)
                 self.oai_response.xml  # try to get xml
```

Ruff (0.15.1)

[warning] Found useless expression. Either assign it to a variable or remove it.

(B018)